PS4k/PS4k.iml

```diff
@@ -14,7 +14,7 @@
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
     </content>
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="1.8" jdkType="JavaSDK" />
```

Comment:
Depends on the features we need and also the aspect of native version you were referring to for the optimizer. I use this version cause the last one installed on my computer.